### ISSUE TITLE
GeoPoint Serialization Helper

### ISSUE THREAD

**Issue by realchrisparker** [elastic/elasticsearch-net#8167]
I'm new to version 8+, and I understand there are no more attributes. However, I can't find any way to set a location so that the maps have a proper geospatial field to work with. No matter what I try, my mapping always ends up like this.
`
"location": {
            "properties": {
              "lat": {
                "type": "float"
              },
              "lon": {
                "type": "float"
              }
            }
          }
`
Could their be an easier way to let the index know this property is a geopoint?

**Describe the solution you'd like**
Whatever works in accordance with the new direction the .net library is taken, it's easy to understand and implement.

**Describe alternatives you've considered**
GeoLocation doesn't have a constructor. No longer has [GeoPoint] attributes. LatLonGeoLocation as a property type doesn't work either. Not sure what else to try.

**Additional context**
None

**Comment by realchrisparker** [comment]
Also tried
`
[JsonProperty( "location" )]
public GeoLocation Location
{
    get
    {
            double[] locs = new double[ 2 ];
            locs[0] = Latitude;
            locs[1] = Longitude;
            var geo = GeoLocation.Coordinates( locs );
            return geo;
    }
}
`
It is definitely possible I am missing something obvious.

**Comment by flobernd** [comment]
Hi @realchrisparker,

you can create the desired mapping like this:

```csharp
await client.Indices.PutMappingAsync<MyClass>("index", m => m.Properties(p => p.GeoPoint(g => g.MyGeoProperty)));
```

Does that answer your question?

**Comment by realchrisparker** [comment]
@flobernd Thank you for your response. That didn't work but I found a stack overflow article and create the index using fluent mapping. In previous versions I used attribute mapping, changing over allowed me to create the index the way I wanted. Thank you for your help and for anyone that runs into the same learning curve, here is the SO https://stackoverflow.com/questions/75842519/how-to-create-index-mapping-in-elastic-clients-elasticsearch-for-net
